mono/mini/tramp-amd64.c

```diff
@@ -520,9 +520,9 @@ mono_arch_create_generic_trampoline (MonoTrampolineType tramp_type, MonoTrampInf
 	 */
 	if (aot) {
 		/* Not really a jit icall */
-		code = mono_arch_emit_load_aotconst (buf, code, &ji, MONO_PATCH_INFO_JIT_ICALL_ADDR, "throw_exception_addr");
+		code = mono_arch_emit_load_aotconst (buf, code, &ji, MONO_PATCH_INFO_JIT_ICALL_ADDR, "rethrow_exception_addr");
```

Comment:
I think you need a matching change in the AOT runtime. 
 Somewhere near https://github.com/mono/mono/blob/a7b9128bf12b50a9d2b10f2de15756311d7b9457/mono/mini/aot-runtime.c#L5310-L5311

Reply:
Interesting. It looks like you're right, but I am surprised this didn't cause any failures on my end when doing a fullaot build.